utilities/src/main/java/io/syndesis/qe/bdd/CommonSteps.java

```diff
@@ -149,11 +159,33 @@ public void cleanMySQLServer() {
     @Given("^deploy MySQL server$")
     public void deployMySQLServer() {
         MysqlTemplate.deploy();
-        ;
     }
 
     @Given("^create standard table schema on \"([^\"]*)\" driver$")
     public void createStandardDBSchemaOn(String dbType) {
         new DbUtils(dbType).createSEmptyTableSchema();
     }
+
+    @Given("^allocate new \"([^\"]*)\" database for \"([^\"]*)\" connection$")
+    public void allocateNewDatabase(String dbLabel, String connectionName) {
+
+        dbAllocatorClient.allocate(dbLabel);
+        log.info("Allocated database: '{}'", dbAllocatorClient.getDbAllocation());
+        TestUtils.setDatabaseCredentials(connectionName.toLowerCase(), dbAllocatorClient.getDbAllocation());
+    }
+
+    @Given("^frees allocated \"([^\"]*)\" database$")
+    public void freeAllocatedDatabase(String dbLabel) {
+        Assertions.assertThat(dbAllocatorClient.getDbAllocation().getDbLabel()).isEqualTo(dbLabel);
+        dbAllocatorClient.free();
+    }
+
+    @And("^she waits until \"([^\"]*)\" pod is reloaded$")
```

Comment:
This should conform to our agreed step name conventions, as it's used in UI tests. Please use "wait" keyword for the name (without "she", she was removed :) )